themes/bootstrap3/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -78,6 +78,19 @@
             </ul>
           </div>
         <?php endif; ?>
+        <?php if (!empty($holding['bound_with_records'])): ?>
```

Comment:
Once things are finalized, we should remember to add this to holdingsils/extended.phtml as well to keep the two versions in sync with each other.

Reply:
Done.  I didn't realize extended.phtml was a thing!  Nice how it's full-width there so no two-column problems.